### DISCUSSION TITLE
Welcome to doughnut Discussions!

### DISCUSSION THREAD

**Discussion by yeongsheng-tan** [nerds-odd-e/doughnut#2] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ngochaho** [comment]
Seeing the number of commits the instructors made to reverse the damages caused during Feb15-Feb19...

**Comment by terryyin** [comment]
Yeah, our turn to learn from your code:)
